libvcx/src/api_lib/api_handle/out_of_band.rs

```diff
@@ -102,7 +102,9 @@ pub fn append_service(handle: u32, service: &str) -> VcxResult<()> {
 pub fn append_service_did(handle: u32, did: &str) -> VcxResult<()> {
     trace!("append_service_did >>> handle: {}, did: {}", handle, did);
     let mut oob = OUT_OF_BAND_SENDER_MAP.get_cloned(handle)?;
-    oob = oob.clone().append_service(&ServiceResolvable::Did(Did::new(did)?));
+    let mut did_sov = "did:sov:".to_owned();
```

Comment:
Let's please leave caller of `append_service_did` to decide on format. With this change, we are forcing caller to supply the legac / non-prefixed format, and do some work for him and it's not obvious. 

Also this is forces OOB sender to dispatch `did:sov` prefixed DID, which wouldn't be understood on aries-vcx clients which have not yet been updated to  revision containing this commit. The suggestion above also solves this problem, as OOB sender can decide himself to use non/prefixed DIDs in OOBs.

Consequently please adjust `test_build_oob_sender_append_services` (or create new test) to cover both cases -when OOB sender produces prefixed/non-prefixed DID.

Reply:
IIt makes perfect sense what you said, so I think leaving the method the same as before, and creating a new test to add the did:sov.